packages/node-config-provider/src/configLoader.spec.ts

```diff
@@ -25,7 +25,7 @@ describe("loadConfig", () => {
     (fromSharedConfigFiles as jest.Mock).mockReturnValueOnce(mockFromSharedConfigFilesReturn);
     const mockFromStatic = "mockFromStatic";
     (fromStatic as jest.Mock).mockReturnValueOnce(mockFromStatic);
-    const envVarSelector = (env: NodeJS.ProcessEnv) => env["AWS_CONFIG_FOO"];
+    const envVarSelector = (env: Record<string, string | undefined>) => env["AWS_CONFIG_FOO"];
```

Comment:
I agree with the suggested change. Would you be able to please put some comments about why this change is being done?. For example, you can mention that NodeJS.ProcessEnv causes types error on non node environments.

Reply:
I did another commit, and added comments to the code. I think that was what you meant, or did you mean comments to the PR?

Reply:
Yeah, that is what I meant.

Thanks!